src/main/java/com/github/fge/jackson/JsonNumEquals.java

```diff
@@ -41,21 +41,50 @@
  * kind of equality.</p>
  */
 public final class JsonNumEquals
-    extends Equivalence<JsonNode>
 {
```

Comment:
Removing this extension seems like a breaking change.

Reply:
Why?
Do you believe anyone relies in it in hierarchy while working with `JsonNumEquals`?

Reply:
I don't believe anything either way. I do know I cannot survey all users of the library. I do know it's part of the surface, and any user could have legitimately started relying on it. By semver conventions, that implies removing it is a breaking change and would necessitate a major version bump.

I'm also not averse to major version bumps for such changes, but I'm apparently an outlier, as others want the major version to be more associated with a collection of features than with API compat. As the new kid on this block, I'm going to defer to the more senior maintainers, if they express a preference.

Reply:
Let me start looking at it. I will today/tomorrow.

Reply:
No rush. Just to expectation set, as soon as I have permissions, I'm going to cut a 1.10 release with just the dependency version bumps. If this commit is accepted, it'll be in a following release.

Reply:
@Capstan I am confused a bit. But I didn't remove that I shaded it? Which changes were you looking at. Basically it was all good and you consern already was resolved originally. I will squash it.

Reply:
I also probably need to make sure that at least the dependent projects in the java-json-tools don't rely on this. Not sure exactly best way to do that without using snapshot releases.